CleanProFinder/CleanProFinder.Mobile/ViewModels/ServiceUserAddPremiseViewModel.cs

```diff
@@ -0,0 +1,53 @@
+﻿using CommunityToolkit.Mvvm.ComponentModel;
+using CleanProFinder.Mobile.Services;
+using CleanProFinder.Shared.ServiceResponseHandling;
+using CommunityToolkit.Mvvm.Input;
+
+namespace CleanProFinder.Mobile.ViewModels;
+
+public partial class ServiceUserAddPremiseViewModel : ObservableObject
+{
+    private readonly IDialogService _dialogService;
+    private readonly IUserPremiseService _userPremisesService;
+
+    public ServiceUserAddPremiseViewModel(IDialogService dialogService, IUserPremiseService userPremisesServic)
```

Comment:
Fix premise service parameter name (in other files too)

Reply:
Fixed